src/main/java/by/exadel/internship/dto/internship/BaseInternshipDTO.java

```diff
@@ -19,7 +19,7 @@
 @SuperBuilder
 @AllArgsConstructor
 @NoArgsConstructor
-public class GuestShortInternshipDTO {
+public class BaseInternshipDTO {
```

Comment:
If it is not used directly make it abstract

Reply:
we use it directly